CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## [14.1.1](https://github.com/theforeman/puppet-foreman_proxy/tree/14.1.1) (2020-09-23)
+
+[Full Changelog](https://github.com/theforeman/puppet-foreman_proxy/compare/14.0.2...14.1.0)
```

Comment:
I think this needs to be `14.1.0...14.1.1`

Reply:
Nice catch, thanks. Updated, and the dates as well.